tests/Data/Food/RecipeTest.elm

```diff
@@ -168,10 +168,10 @@ suite =
 
                             Ok scoring ->
                                 [ Unit.impactToFloat scoring.all
-                                    |> Expect.within (Expect.Absolute 0.01) 478.0375489673356
+                                    |> Expect.within (Expect.Absolute 0.01) 478.03754896733557
                                     |> asTest "should properly score total impact"
                                 , Unit.impactToFloat scoring.allWithoutComplements
-                                    |> Expect.within (Expect.Absolute 0.01) 476.8371815514294
+                                    |> Expect.within (Expect.Absolute 0.01) 476.83718155142935
```

Comment:
`Expect.Absolute 0.01` means we have a `0.01` tolerance for deviations. Why did you bother updating these two values which are both well within this tolerance? You were getting test failures here?

Reply:
Forgot why. The probable reason is I once got a very different result which made me update the test, then got back a similar one which would not have make the test fail